components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -433,3 +436,25 @@ message WorkspaceClass {
     // is_default indicates if this workspace class is the default one
     bool is_default = 4;
 }
+
+message GetDefaultWorkspaceImageRequest {
+    optional string workspace_id = 1;
```

Comment:
Maybe also allow users to pass in an org ID for querying?

Reply:
Maybe wait until somewhere needs it? (Dashboard is using server getOrgSettings to specify get org default workspace image now)